### DISCUSSION TITLE
"Preview" of CMS Strapi example works different from other CMS examples

### DISCUSSION THREAD

**Discussion by msreekm** [vercel/next.js#14683] [Help]
It maybe good to update the CMS Strapi readme to clarify that if you update an existing post and put it in draft mode , this post will disappear from the website, in other CMS's like prisma, DotoCms will show the last published post . works well only for new posts in draft! 

In strapi published is a custom field sort of like a hack  whereas other CMS has support for this out of the box?

**Comment by SalahAdDin** [comment]
No any feedback here, right?

**Comment by SalahAdDin** [comment]
Could you make this example work? I cannot.
